doc: added details around console.timeEnd changes (#35027)
nodejs/node: doc/api/console.md

Starts a timer that can be used to compute the duration of an operation. Timers
are identified by a unique `label`. Use the same `label` when calling
[`console.timeEnd()`][] to stop the timer and output the elapsed time in

> suitable timeunits to `stdout`. For example, if the elapsed

Reviewer: ```suggestion
suitable time units to `stdout`. For example, if the elapsed
```

Author: Done 👍

> suitable timeunits to `stdout`. For example For example, if the elapsed

Reviewer: ```suggestion
suitable timeunits to `stdout`. For example, if the elapsed
```

> time is 3869ms, `console.timeEnd()` now displays "3.869s". Timer

Reviewer: ```suggestion
time is 3869ms, `console.timeEnd()` displays "3.869s". Timer
```

Author: Done 👍

> time is 3869ms, console.timeEnd() now displays "3.869s". Timer

Reviewer: ```suggestion
time is 3869ms, `console.timeEnd()` now displays "3.869s". Timer
```

> durations are accurate to the sub-millisecond.

Reviewer 2: Small side note, is this supposed to read “to the millisecond”? It doesn’t really make sense to say that there might be more granularity than that if the timer doesn’t actually expose it, right?

Reviewer: @yashLadha can you please also fix this?

Author: Done 👍  Removed the un-necessary granularity details.
